classes/users.py

```diff
@@ -0,0 +1,68 @@
+# This file will registration and retrieval of the users details
+import re
+users = {}
+
+
+class Users(object):
+    """This class will handle all the functions related to the users"""
+
+    def __init__(self, username=None, email=None, password=None):
+        """constructor to initialize the global variables"""
+        self.username = username
+        self.email = email
+        self.password = password
+
+    def register_user(self, username, email, password, confirm_password):
+        """
+        This method will validate and verify the user details before storing
+        """
+        nonspace_username = re.sub(r'\s+', ' ', username).strip()
+        regusername = "[a-zA-Z0-9- .]"
+        regemail = r"([a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)"
+        if re.match(regusername, nonspace_username):
```

Comment:
From line 22 to 41 those are too many `if` statements :flushed: please see how you can refactor that.